source/MissionAction.cpp

```diff
@@ -296,9 +311,14 @@ void MissionAction::Save(DataWriter &out) const
 		}
 		if(!conversation.IsEmpty())
 			conversation.Save(out);
-		
-		for(const auto &it : gifts)
+		for(const auto &it : giftShips)
+		{
+			out.Write("ship", it.first->Name(), it.second);
+		}
+		for(const auto &it : giftOutfits)
+		{
 			out.Write("outfit", it.first->Name(), it.second);
+		}
```

Comment:
Style: unnecessary braces.
```suggestion
			out.Write("ship", it.first->Name(), it.second);
		for(const auto &it : giftOutfits)
			out.Write("outfit", it.first->Name(), it.second);
```